spec/requests/security_activity_spec.rb

```diff
@@ -40,13 +54,25 @@
     expect_event SecurityActivity::LOGIN_SUCCESS
   end
 
+  it "records LOGIN_SUCCESS event with additional analytics data from confirmation email" do
```

Comment:
It would be good to also see a feature spec for this event, just to confirm the parameter is passed around properly.

Reply:
Good idea, I've added a feature test covering all the security activities we log this foe